WordPress/Classes/Services/AppUpdate/AppUpdateCoordinator.swift

```diff
@@ -0,0 +1,107 @@
+import Foundation
+
+enum AppUpdateType {
```

Comment:
(nit) It's probably better modeled as:

```swift
struct AppUpdateInfo {
   var appInfo: AppStoreInfo
   var isRequired: Bool
}
```

Reply:
Done! [89e1488](https://github.com/wordpress-mobile/WordPress-iOS/pull/23195/commits/89e1488a2a708f08b05aef3616a44c90571f802f)